.appveyor.yml

```diff
@@ -54,84 +54,84 @@ environment:
       PYVER: 27
```

Comment:
isn't it time to drop this one, perhaps replace with `PYVER: 38`

Reply:
I'd like to resolve this taskmastertrace error. Maybe push a 3.1.3 then start removing py27 items. (or minimally just error out if run on py27)
With hope to release a 4.0.0 on 1/1/2020

Reply:
Whichever way works.  I won't push any drop-2.7 suggestions until the builders aren't building for 2.7, but if you want to defer that event, it's fine.  Jan 1 feels a bit early to me... even pdotorg is planning another 2.7 release in 2020:

> Planned future release dates:

    2.7.18 code freeze January, 2020
    2.7.18 release candidate early April, 2020
    2.7.18 mid-April, 2020

More interesting would be what support promise would go on scons.org: stick with 3.1 if you cannot get off Python 2.7.  3.1 will/will not receive updates in case of serious errors - that sort of thing.